### DISCUSSION TITLE
custom login by rest api with jwt

### DISCUSSION THREAD

**Discussion by Pger-Y** [Budibase/budibase#9213] [Help] [answered]
Hi,
 Is it possible to login by my backend server and set jwt token for the reset query? I have read https://github.com/Budibase/budibase/discussions/3214 it is sad custom login api feature is not supported and low priority,in consideration of the discuss happened at Nov 1, 2021,is this feature suppoeded yet ? I have read the doc but there is nothing about this.Does anybody have the same requirement or know the progess. thanks :)

**Comment by melohagan** [comment, marked as answer]
Hey @Pger-Y 

This feature is not supported yet, and there are no plans in the intermediate future.

**Reply by Pger-Y** [reply]
OK，Thanks~ compared budibase/appsmith/tooljet,I like budibase most~It's really a pity that I can't use it in my situation,maybe in the future after you supported this feature~ have a nice day :)

**Reply by Pger-Y** [reply]
Oh,I just have an idea , is it possible that I sync my user data like name/role/password to budibase and log in,after that query outer rest api with these info like userid/role~

**Reply by eftomi** [reply]
I tried something similar ...

https://github.com/Budibase/budibase/discussions/8824

**Reply by Pger-Y** [reply]
thanks @eftomi I'll check it out~
